src/swc.ts

```diff
@@ -21,6 +26,11 @@ const completionSpec: Fig.Spec = {
       name: "--env-name",
       description:
         "The name of the 'env' to use when loading configs and plugins. Defaults to the value of SWC_ENV, or else NODE_ENV, or else development",
+      args: {
+        name: "ENV_NAME",
```

Comment:
Probably we can provide suggestions for common environment names here like 
```suggestion
        name: "ENV_NAME",
        suggestions: ["production", "development", "test"],
```